### PR TITLE
Fix cli args help, align vars order

### PR DESCRIPTION
**Pull request by sponomarev**
- Add deprecation warning for default interface
- CLI args typo is fixed
- Order of CLI vars is aligned with docs for readability

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -103,7 +105,7 @@ USAGE
   anycable-go [options]
 
 OPTIONS
-  --host                   Server host, default: localhost, env: ANYCABLE_HOST
```
Comment (sponomarev) [inline cli/cli.go]: I feel the necessity to change default instead of the help but it's too dangerous 🤷‍♂️.

File: cli/cli.go [hunk 2]
```diff
@@ -44,33 +44,35 @@ func init() {
 	}
 
 	// Config vars
-	fs.StringVar(&defaults.RPCHost, "rpc_host", "localhost:50051", "")
```
Comment (sponomarev) [inline cli/cli.go]: These lines are rearranged only according to the logical groups and order presented in Help

File: cli/cli.go (hunk 1 above)
Comment (palkan) [reply cli/cli.go]: Mm, then we shouldn't change the help message.

Can we add a deprecation message?

For example:
- use some "fake" value as the default, e.g. (in https://github.com/anycable/anycable-go/blob/85d3347a2e0943e12259c530abb0be2b317f5f3a/cli/cli.go#L50):

```go
fs.StringVar(&defaults.Host, "host", "0.0.0.0|deprecated", "")
```

- in `config.ensureParsed` check whether the value is equal to the fake one, show warning and restore it back to `"0.0.0.0"`
- in the help show `"default: 0.0.0.0 (deprecated, will be changed to "localhost" )`

Comment (sponomarev) [reply cli/cli.go]: Sounds good to me. I was waiting for this suggestion.

**Pull request merged**